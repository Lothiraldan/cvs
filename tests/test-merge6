#!/bin/sh -x

cat <<'EOF' > merge
#!/bin/sh
echo merging for `basename $1`
EOF
chmod +x merge
export HGMERGE=./merge

mkdir A1
cd A1
hg init
echo This is file foo1 > foo
echo This is file bar1 > bar
hg add foo bar
hg commit -m "commit text" -d "0 0"

cd ..
hg clone A1 B1

cd A1
rm bar
hg remove bar
hg commit -m "commit test" -d "0 0"

cd ../B1
echo This is file foo22 > foo
hg commit -m "commit test" -d "0 0"

cd ..
hg clone A1 A2
hg clone B1 B2

cd A1
hg pull ../B1
hg update -m
hg commit -m "commit test" -d "0 0"
echo bar should remain deleted.
hg manifest

cd ../B2
hg pull ../A2
hg update -m
hg commit -m "commit test" -d "0 0"
echo bar should remain deleted.
hg manifest
