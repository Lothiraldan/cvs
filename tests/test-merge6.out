updating to branch default
2 files updated, 0 files merged, 0 files removed, 0 files unresolved
updating to branch default
1 files updated, 0 files merged, 0 files removed, 0 files unresolved
updating to branch default
2 files updated, 0 files merged, 0 files removed, 0 files unresolved
pulling from ../B1
searching for changes
adding changesets
adding manifests
adding file changes
added 1 changesets with 1 changes to 1 files (+1 heads)
(run 'hg heads' to see heads, 'hg merge' to merge)
1 files updated, 0 files merged, 0 files removed, 0 files unresolved
(branch merge, don't forget to commit)
bar should remain deleted.
f9b0e817f6a48de3564c6b2957687c5e7297c5a0 644   foo
pulling from ../A2
searching for changes
adding changesets
adding manifests
adding file changes
added 1 changesets with 0 changes to 0 files (+1 heads)
(run 'hg heads' to see heads, 'hg merge' to merge)
0 files updated, 0 files merged, 1 files removed, 0 files unresolved
(branch merge, don't forget to commit)
bar should remain deleted.
f9b0e817f6a48de3564c6b2957687c5e7297c5a0 644   foo
